### ISSUE TITLE
Laravel 9 issue

### ISSUE THREAD

**Issue by wontone18** [jgrossi/corcel#604]
- Corcel Version: 5.0
- Framework Name & Version: 9.0
- PHP Version: 8.1
- Database Driver & Version: 

### Description: 
       Problem 1
    - Root composer.json requires jgrossi/corcel 5.0 -> satisfiable by jgrossi/corcel[v5.0.0].
    - jgrossi/corcel v5.0.0 requires illuminate/database ^8.0 -> found illuminate/database[v8.0.0, ..., 8.x-dev] but it conflicts with your root composer.json require (^9.0).

### Steps To Reproduce:

Its not working with laravel 9.0. I am thinking it creating conflict with the database.

**Comment by bumbummen99** [comment]
https://github.com/corcel/corcel/pull/603

`jgrossi/corcel v5.0.0 requires illuminate/database ^8.0`

^8.0 does include every upper version until excluding 9.0

**Comment by wontone18** [comment]
> #603
> 
> `jgrossi/corcel v5.0.0 requires illuminate/database ^8.0`
> 
> ^8.0 does include every upper version until excluding 9.0

have u try it with laravel 9?

**Comment by bumbummen99** [comment]
> > #603
> > `jgrossi/corcel v5.0.0 requires illuminate/database ^8.0`
> > ^8.0 does include every upper version until excluding 9.0
> 
> have u try it with laravel 9?

Yes i did just for you now and it is working fine. Please read my last comment for information on what the actual issue you have included in the OP is. You can get more information here:
https://semver.org/
https://getcomposer.org/doc/articles/versions.md

**Comment by wontone18** [comment]
sure i will check it tonight

**Comment by wontone18** [comment]
> > > #603
> > > `jgrossi/corcel v5.0.0 requires illuminate/database ^8.0`
> > > ^8.0 does include every upper version until excluding 9.0
> > 
> > 
> > have u try it with laravel 9?
> 
> Yes i did just for you now and it is working fine. Please read my last comment for information on what the actual issue you have included in the OP is. You can get more information here: https://semver.org/ https://getcomposer.org/doc/articles/versions.md

My php version is 8.1.1 and amazing how it works in your side because illuminate/database:^8.0 only support php ^7.3. And latest "illuminate/database": "^9.0" works with php 8.1.1. 


 ```
Problem 1
    - jgrossi/corcel[v5.1.0, ..., v5.1.1] require illuminate/database ^8.0 -> found illuminate/database[v8.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.
    - Root composer.json requires jgrossi/corcel ^5.1 -> satisfiable by jgrossi/corcel[v5.1.0, v5.1.1].

You can also try re-running composer require with an explicit version constraint, e.g. "composer require jgrossi/corcel:*" to figure out if any version is installable, or "composer require jgrossi/corcel:^2.1" if you know which you need.
```

**Comment by bumbummen99** [comment]
Of course you have to raise the version constraint for all illuminate packages to also include ^9.0 in composer.json of this package, this is what i tried to explain in my answers to your issue. You will have to use the version provided in #603 or do your own fork until this has been merged. Of course it does not work with Laravel 9, the version constraint does restrict that it as i have explained - that is totally intended as L9 could be  a breaking change.

What you said about illuminate/database is not true, please look at https://packagist.org/packages/illuminate/database#v8.12.0

To verify yourself that it does work, simply clone this repository, edit composer.json and raise all illuminate to ^9 and testbench to 7, then composer install and run `php vendor/bin/phpunit` and tests will run just fine.

**Comment by bumbummen99** [comment]
Also this issue is a duplicate of #602

**Comment by wontone18** [comment]
> Of course you have to raise the version constraint for all illuminate packages to also include ^9.0 in composer.json of this package, this is what i tried to explain in my answers to your issue. You will have to use the version provided in #603 or do your own fork until this has been merged. Of course it does not work with Laravel 9, the version constraint does restrict that it as i have explained - that is totally intended as L9 could be a breaking change.
> 
> What you said about illuminate/database is not true, please look at https://packagist.org/packages/illuminate/database#v8.12.0
> 
> To verify yourself that it does work, simply clone this repository, edit composer.json and raise all illuminate to ^9 and testbench to 7, then composer install and run `php vendor/bin/phpunit` and tests will run just fine.

Thanks its done with the manual way. i also Updated my composer/autoload_psr4.php and installed.json( here also i updated versions) then composer dump-autoload.

i hope you will come with the new version soon

**Comment by jgrossi** [comment]
hey 👋 support to Laravel 9 included on the new tag `v6.0.0`
